### PR TITLE
feat:add product-listing page

### PR DESCRIPTION
**Pull request by shruthipatil11**
added product listing page

### CODE REVIEW

File: src/app/product-listing/product-listing.html [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+                    </ul>
+                </div>
+            </div>
+        </aside>
+        <main>
+            <div class="mt-mb-1"><span class="font-weight-600">Showing all products</span> (showing 20 products)
+            </div>
+            <section class="d-flex-wrap">
+                <!-- Snippet for card -->
+                <div class="card card__badge card--vertical">
```
Comment (mayuresh-tech) [inline src/app/product-listing/product-listing.html:92]: Good usage of vertical and horizontal cards!
Clean code

File: src/app/product-listing/product-listing.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+.filters__range {
+    width: 100%;
+}
+
+.filters h3 {
+    font-weight: 600;
+    margin-bottom: 1rem;
+}
+.filters input {
+    margin-right: 0.5rem;
```
Comment (sagargse) [inline src/app/product-listing/product-listing.css]: For less values use px instead of rem

**Review by sagargse — APPROVED** [review]
rest all looks good to me

File: src/app/product-listing/product-listing.css (hunk 2 above)
Comment (shruthipatil11) [reply src/app/product-listing/product-listing.css]: done

**Review by mayuresh-tech — COMMENTED** [review]
Good to merge

**Pull request merged**